.github/workflows/ci_cron.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     runs-on: ubuntu-22.04
     steps:
       - uses: actions/checkout@v2
-      - uses: actions/setup-ruby@v1
+      - uses: ruby/setup-ruby@v1
```

Comment:
Did you know of this change because there was a warning that said "actions/setup-ruby@v1 is deprecated, use ruby/setup-ruby@v1"?

Reply:
Yep! This is task 1 of the 98 tasks set forth by #1317.

Reply:
Thank you :)